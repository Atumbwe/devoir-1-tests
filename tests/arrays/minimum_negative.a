-423 -5 -3426 323 -564 423 -53 -324 4234 -634 -53 -234 min