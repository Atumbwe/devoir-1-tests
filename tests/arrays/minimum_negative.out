 _______
< -3426 >
 -------
        \   ^__^
         \  (oo)\_______
            (__)\       )\/\
                ||----w |
                ||     ||
